app/src/main/java/org/gnucash/android/service/ScheduledActionService.java

```diff
@@ -175,14 +175,32 @@ private static int executeBackup(ScheduledAction scheduledAction, SQLiteDatabase
         try {
             //wait for async task to finish before we proceed (we are holding a wake lock)
             new ExportAsyncTask(GnuCashApplication.getAppContext(), db).execute(params).get();
-            scheduledAction.setExecutionCount(++executionCount);
+            ++executionCount;
         } catch (InterruptedException | ExecutionException e) {
             Crashlytics.logException(e);
             Log.e(LOG_TAG, e.getMessage());
         }
+
+        // HACK: Add missed runs to executionCount so computeNextScheduledExecutionTime()
```

Comment:
Yes, my initial intention was to separate scheduled exports and transactions into different classes but again, it depended on having an XML pull parser for importing. That's why I did this temporal hack. So I guess I'll start with the parser as soon as I fix these bugs.